ruby/vampires.rb

```diff
@@ -0,0 +1,63 @@
+puts "How many employees are there ?"
+employees=gets.chomp.to_i
+for i in 1..employees do
+  puts "What is your name?"
+  name = gets.chomp
+  puts "How old are you?"
+  age = gets.chomp.to_i
+  puts "What year were you born?"
+  birth_year = gets.chomp.to_i
+  puts "Our company cafeteria serves garlic bread. Should we order some for you?(y/n)"
+  garlic = gets.chomp
+  puts "Would you like to enroll in the company's health insurance?(y/n)"
+  insurance = gets.chomp
+
+  # p name, age, birth_year, garlic, insurance
+
+  probably_not_a_vampire=false;

```

Comment:
Instead of creating multiple variables for the end result of the vampire, we can just set one variable and reassign the value of it based on the results of each conditional statement. something like this
```
if condition_1
	result = 'low'
end

if condition_2
	result = 'medium'
end

if condition_3
	result = 'high'
end

puts result
```